src/components/NavBar/index.tsx

```diff
@@ -1,69 +1,66 @@
 import React from 'react';
 import { Colors, Styles } from '../../constants';
-import { MainMenu, IconProps } from './MainMenu';
+import { MainMenu, IconProps, IconStyle } from './MainMenu';
 import { SbNavLink, SbNavlinkProps } from '../SbNavLink';
 import { SearchBar } from '../SearchBar';
 import { HelpCircle, Menu } from 'react-feather';
+import MediaQuery from 'react-responsive';
+import { Link } from 'react-router-dom';
 
 /*tslint:disable: no-require-imports no-var-requires */
-const sbLogo: string = require('@sbac/sbac-ui-kit/src/images/SmarterBalanced-Logo.png') as string;
+const sbLogo = require('@sbac/sbac-ui-kit/src/images/SmarterBalanced-Logo.png') as string;
 
 export interface NavBarProps {
-  links?: SbNavlinkProps[];
   siteName: string;
   mainContentId: string;
 }
 
-export const Logo = (): JSX.Element => (
-  <div className="nav-titleGroup-item">
-    <a
-      rel="noopener noreferrer"
-      target="_blank"
-      href="http://www.smarterbalanced.org/"
-      title="Smarter Balanced Home"
-    >
-      <img alt="Smarter Balanced Logo" src={sbLogo} />
-    </a>
+export const HeaderLogo: React.SFC = () => (
+  <a href={'https://www.smarterbalanced.org/'}>
+    <img alt="Smarter Balanced logo" src={sbLogo} className="sb-logo" />
     <style jsx>{`
-      .nav-titleGroup {
-        display: flex;
-        flex-flow: row;
-        align-items: center;
-        flex-grow: 1;
+      .sb-logo {
+        width: 128px;
+        padding: 3px ${Styles.paddingUnit} 3px 3px;
       }
-      img {
-        width: 183px;
-        padding-right: ${Styles.paddingUnit};
+
+      @media (max-width: 500px) {
+        .sb-logo {
+          width: 100px;
+        }
       }
     `}</style>
-  </div>
+  </a>
 );
 
-export const NavBar = (props: NavBarProps): JSX.Element => {
-  const IconAttributes: IconProps = {
-    size: 40,
-    color: Colors.sbGray
+export const NavBar: React.SFC<NavBarProps> = props => {
+  const noUnderline: React.CSSProperties = {
+    textDecoration: 'none'
   };
-  const help = <HelpCircle {...IconAttributes} />;
-  const menu = <Menu {...IconAttributes} />;
 
   return (
     <header>
       <nav className="nav-container">
         <div className="left-content">
-          <Logo />
-          <MainMenu />
+          <HeaderLogo />
+          <MediaQuery minWidth={715}>
+            <MainMenu />
+          </MediaQuery>
         </div>
         <div className="right-content">
-          <span>
+          <span className="right-spacing grow-search">
             <SearchBar />
           </span>
-          <span>
-            <SbNavLink url={'placeholder'} content={help} />
-          </span>
-          <span>
-            <SbNavLink url={'placeholder'} content={menu} />
+          <span className="right-spacing">
+            <Link to="placeholder" style={noUnderline}>
+              <HelpCircle {...IconStyle} />
+            </Link>
           </span>
+          <MediaQuery minWidth={715}>
```

Comment:
This number needs to be recorded in the `constants` file. We don't want our media query breakpoints to fragment all over the place.